client/src/components/QandA/AddQuestion.jsx

```diff
@@ -0,0 +1,74 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+
+class AddQuestion extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      questionEntry: '',
+      nicknameEntry: '',
+      emailEntry: '',
+    };
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleChange(e) {
+    if (e.target === 'question') { // determine distinguishing target value for given input
+      this.setState({questionEntry: e.target.value});
+    }
+    if (e.target === 'nickname') {
+      this.setState({nicknameEntry: e.target.value});
+    }
+    if (e.target === 'email') {
+      this.setState({emailEntry: e.target.value});
+    }
+  }
+
+  handleSubmit(e) {
+    e.preventDefault();
+    // ensure that both question and email fields have been filled out and are valid formats via helper functions
+    // when API is implemented, use axios POST request to submit data to API
+  }
+
+  render() {
+    return (
+      <div>
+        <button type="button" className="btn btn-light border-4 border-dark" data-bs-toggle="modal" data-bs-target="#addQuestionModal">Add A Question +</button>
+        <div className="modal fade" id="addQuestionModal" tabIndex="-1" aria-labelledby="addQuestionModalLabel" aria-hidden="true">
+          <div className="modal-dialog modal-dialog-scrollable">
+            <div className="modal-content">
+              <div className="modal-header">
+                <h5 className="modal-title" id="addQuestionModalLabel">Ask Your Question - </h5>
+                <h6 className="modal-title" id="addQuestionModalLabel">About the {this.props.currentProductName}</h6>
+                <button type="button" className="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
```

Comment:
This is actually part of the Bootstrap functionality that allows the overall modal-div above this (on line 38) to be 'selected' by the labels here, which are given as IDs. Changing this might break the modal from what I understood.

Reply:
I see.
If it's working correctly, I guess bootstrap is overriding some CSS/HTML rules.